samples/boards/nrf52/mesh/onoff-app/prj.conf

```diff
@@ -2,6 +2,14 @@
 CONFIG_MAIN_STACK_SIZE=512
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
 
+CONFIG_BT_SETTINGS=y
+CONFIG_FLASH=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_FLASH_MAP=y
+CONFIG_FCB=y
+CONFIG_SETTINGS=y
+CONFIG_SETTINGS_FCB=y
```

Comment:
Please split this into a separate PR (that's what I intended when I asked for this addition). It can then probably be merged must faster than this PR, since I don't see anything wrong with the patch. If any of this feels like a lot of work, please get familiar with `git rebase -i`. That will get the work done in a matter of seconds, or a minute at most.